A/Antlr4CppRuntime/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Antlr4CppRuntime"
+version = v"4.9.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/antlr/antlr4/archive/refs/tags/$(version).tar.gz", "efe4057d75ab48145d4683100fec7f77d7f87fa258707330cadd1f8e6f7eecae")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/antlr4-4.9.3/runtime/Cpp
+cmake . -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+make
```

Comment:
Make it more future-proof by removing the reference to the version, build in a subdirectory and enable parallel build
```suggestion
cd $WORKSPACE/srcdir/antlr4*/runtime/Cpp
mkdir build && cd build
cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
make -j${nproc}
```